conf/logging.cfg

```diff
@@ -1,5 +1,5 @@
 [handlers]
-keys=consoleHandler
+keys=consoleHandler, fileHandler
```

Comment:
My bad, I misinterpret the handlers sections. I thought it will just define a handler and won't use it until some logger use the handler.

I'll remove this part. Nice catch.